setup_instructons/wsl.md

```diff
@@ -104,27 +112,27 @@ You can install the Windows Terminal app from the Microsoft Store: [App Store Pa
    rosdep update
    ```
 
-1. Run rosdep
+2. Run rosdep
```

Comment:
For markdown numbered lists, it's usually preferred to use automatic numbering (where the raw text has all of the numbers as 1) so that the numbering stays consistent if items are re-ordered, added, or removed.

Reply:
didn't know that about markdown, that's cool!